login/service.go

```diff
@@ -339,7 +339,7 @@ func (keycloak *KeycloakOAuthProvider) checkFederatedIdentity(ctx context.Contex
 		log.Error(ctx, map[string]interface{}{
 			"err": err.Error(),
 		}, "Unable to crete http request")
-		return false, er.NewInternalError(errs.Wrap(err, "unable to crete http request"))
+		return false, er.NewInternalError(ctx, errs.Wrap(err, "unable to crete http request"))
```

Comment:
Again, purpose of this PR is not to fix typos but fixed in e0d4eca5e1a5ac1e02530e6bafee8ada3b5e5e34